domains/kylemartin

```diff
@@ -0,0 +1,10 @@
+{
+  "description": "Kyle Martin",
+  "repo": "https://github.com/js-kyle/js-kyle.github.io",
+  "owner": {
+    "username": "js-kyle"
```

Comment:
i would like to see an email here. if you dont have, or dont want to leak, just leave it `{"email": ""}`.
`